### ISSUE TITLE
[BUG] Unexpected serialization of maps in informal trace format

### ISSUE THREAD

**Issue by rnbguy** [informalsystems/apalache#1293]
## Description

Singleton maps are deserialized to `#map: [ "key": "value" ]` instead of `#map: [ [ "key": "value" ] ]` in informal trace format.

Is this expected? I don't see any mention at [ADR-015](https://apalache.informal.systems/docs/adr/015adr-trace.html).

## Example

```
"#map": [
  "cosmoshub",
  0
]
```

## Expected behavior

```
"#map": [
  [
    "cosmoshub",
    0
  ]
]
```

**Comment by konnov** [comment]
That's a bug. Fixed in #1296. Check and approve.

**Comment by rnbguy** [comment]
I checked the build. It works :+1: thanks
